spaces/safe2/index.json

```
@@ -0,0 +1,24 @@
+{
+  "token": "0x250a3500f48666561386832f1F1f1019b89a2699",
+  "name": "SAFE2",
+  "network": "1",
+  "symbol": "SAFE2",
+  "skin": "safe2",
```

To use the existing `safe` skin change skin field value and delete `safe2.scss`.
```suggestion
  "skin": "safe",
```

---

Changed